### PR TITLE
MTM-49614 update screenshots for audits, and web applications .…

### PR DESCRIPTION
**Pull request by jkot-softwareag**
… for 10.17 release. Minor fixes in text.

### CODE REVIEW

File: content/users-guide/administration-bundle/managing-applications.md [hunk 1]
```diff
@@ -205,9 +197,6 @@ Duplicating an application might be useful if you want to customize a subscribed
 2. In the upcoming dialog, select **Duplicate existing application**.
 3. Select the desired application from the dropdown list, for example "Cockpit".
 4. In the next window, provide a name for the application, an application key to identify the application, and a path as part of the URL to invoke the application. Per default, the values of the original application are provided, extended by a number. If you set the path to the path of the original subscribed application, your own application will overrule the subscribed application.
-<br><br>
-<img src="/images/users-guide/Administration/admin-application-duplicate-2.png" alt="Duplicate application">
-<br><br>
```
Comment (jkot-softwareag) [inline content/users-guide/administration-bundle/managing-applications.md:210]: I think one image is enough for duplicating application. For now I preservied admin-application-duplicate-3.png from line 217/206. This one is same, but without the switcher for "Overrule".

File: content/users-guide/administration-bundle/managing-applications.md [hunk 2]
```diff
@@ -332,9 +321,9 @@ Users can restore previous versions of an application from an archive.
 If a hosted application is not deployed correctly, users may reactivate it.
 
 1. Open the application properties for the respective application by clicking on it.
-3. In the **Activity log** section, open the context menu for the desired version by clicking the menu icon and select **Reactivate**.
+3. In the **Activity log** section, open the context menu for the desired version by clicking the menu icon and select **Reactivate archive**.
 
-The selected application will be reactivated by removing the respective files from the application directory and unpacking the host application package again.
+The selected application will be reactivated by removing the respective files from the application directory and unpacking the web application package again.
```
Comment (jkot-softwareag) [inline content/users-guide/administration-bundle/managing-applications.md:326]: it is called "HOSTED" Application or web application, not "host"

**Review by d-mariak — APPROVED** [review]

**Comment by darluk** [comment]
regarding static/images/users-guide/Administration/admin-application-duplicate-3.png there is a different quantity of buttons on the modal   for german and English versions

File: content/users-guide/administration-bundle/managing-applications.md [hunk 3]
```diff
@@ -115,6 +113,7 @@ In the **All applications** tab, subscribed applications are labeled as "Subscri
 <td style="text-align:left">devicemanagement</td>
 <td style="text-align:left">Web application</td>
 <td style="text-align:left">{{< standard-tenant >}}, {{< enterprise-tenant >}}</td>
+</tr>
```
[the remark this answers is not in the document]
Comment (jkot-softwareag) [reply content/users-guide/administration-bundle/managing-applications.md:116]: to not force the browser to fix such mistakes on the fly during rendering ;). Here is a screenshot from dev-tools:
image

File: content/users-guide/administration-bundle/managing-applications.md [hunk 4]
```diff
@@ -45,8 +45,6 @@ Both can be accessed via the **Ecosystem** menu in the navigator.
 
 Additionally, in {{< enterprise-tenant >}}s, it is possible to configure **Default subscriptions**, that means you can specify a list of applications that are subscribed by default to every new tenant on creation and/or to all existing tenants on platform upgrade. For details, see [Enterprise tenant > Default subscriptions](/users-guide/enterprise-tenant/#default-subscriptions).
 
-<img src="/images/users-guide/Administration/admin-menu.png" alt="Applications menu">
```
Comment (karolkaczmarski-sag) [inline content/users-guide/administration-bundle/managing-applications.md:48]: Please apply changes to german version too.

File: content/users-guide/administration-bundle/managing-applications.md (hunk 1 above)
Comment (karolkaczmarski-sag) [reply content/users-guide/administration-bundle/managing-applications.md:210]: I think admin-application-duplicate-3.png is not needed

File: content/users-guide/administration-bundle/managing-applications.md (hunk 4 above)
Comment (BeateRixen) [reply content/users-guide/administration-bundle/managing-applications.md:48]: @karolkaczmarski-sag Modifying the German md files is not required as we create them automatically from the English source files during the translation process. All functional elements will be taken from the English files, we only replace the texts (and modify the paths to the images). So if the link is removed fro the English files, this is sufficient. All text changes will be adapted during translation. Sorry, I should have stated clearer that we only need the German images.

**Review by BeateRixen — APPROVED** [review]

File: content/users-guide/administration-bundle/managing-applications.md (hunk 4 above)
Comment (karolkaczmarski-sag) [reply content/users-guide/administration-bundle/managing-applications.md:48]: @BeateRixen good to know, thanks! I didn't know it's automated process :)

**Review by karolkaczmarski-sag — APPROVED** [review]

**Comment by jkot-softwareag** [comment]
> regarding static/images/users-guide/Administration/admin-application-duplicate-3.png there is a different quantity of buttons on the modal for german and English versions.

Good catch, I have taken screenshot one more time (the previous was from tenant owning application). Fixed and squash-rebased in PR.

File: content/users-guide/administration-bundle/managing-applications.md (hunk 4 above)
Comment (jkot-softwareag) [reply content/users-guide/administration-bundle/managing-applications.md:48]: @BeateRixen should we look for some German version or this is just a matter of translation and images should just have same filenames?

Comment (BeateRixen) [reply content/users-guide/administration-bundle/managing-applications.md:48]: @jkot-softwareag The images should have the exact same file names, just the path is different. I don´t really get what you mean with version?

Comment (jkot-softwareag) [reply content/users-guide/administration-bundle/managing-applications.md:48]: never mind, it was responded in your previous comment :). I have written my question without page refresh and without seeing this comment.

**Pull request merged**